radixdlt/babylon-wallet-ios
# ABW-1536 Show Ledgers from Settings

Jira ticket: [ABW-1536](https://radixdlt.atlassian.net/browse/ABW-1536)

## Description
There is no menu item in settings that shows us the list of connected ledgers. This PR refactors settings and adds this.

## How to test

Connect a Ledger device
Go to settings
Select the Ledger Hardware Wallets option
-> Your connected ledger should appear on screen in a list

## Screenshot
<img src="https://github.com/radixdlt/babylon-wallet-ios/assets/123396602/6c56dab0-5b98-4734-b9f8-946339443e67" width="350" />

## PR submission checklist

[ABW-1536]: https://radixdlt.atlassian.net/browse/ABW-1536?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
